OptionExt.nuspec

```diff
@@ -13,7 +13,7 @@
     <tags>F# Option Computation-Expression</tags>
 
     <dependencies>
-      <dependency id="FSharp.Core" version="4.1.17" />
+      <dependency id="FSharp.Core" version="4.6.2" />
```

Comment:
If we're updating from FSharp.Core 4.1.17 to 4.6.2 (which we really need to do in order to have any sort of support for voptions), we should probably increment the main version number in the nuspec.  I would consider that a breaking change.  (Not all users could be expected to update to the latest version of FSharp.Core)

Reply:
✔️